### PR TITLE
fix: adjust margins error messages in traffic source

### PR DESCRIPTION
**Pull request by GusBaamonde**
messageError

### CODE REVIEW

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js [hunk 1]
```diff
@@ -127,11 +128,11 @@ const ReportsTrafficSources = function({ domainName, dateFrom, dependencies: { d
               </S.ListItem>
             ))
           ) : (
-            <div className="dp-msj-error bounceIn">
+            <ContentError className="dp-msj-error bounceIn">
               <p>
                 <FormattedMessage id="trafficSources.error" />
               </p>
-            </div>
+            </ContentError>
```
Comment (andresmoschini) [inline src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js]: Should you use the `S.`convention here?

CC: @fcoronelmakingsense

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js [hunk 2]
```diff
@@ -127,11 +128,11 @@ const ReportsTrafficSources = function({ domainName, dateFrom, dependencies: { d
               </S.ListItem>
             ))
           ) : (
-            <div className="dp-msj-error bounceIn">
+            <ContentError className="dp-msj-error bounceIn">
```
Comment (andresmoschini) [inline src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js]: These classes should be also part of the styled component, right?

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js [hunk 3]
```diff
@@ -127,11 +128,11 @@ const ReportsTrafficSources = function({ domainName, dateFrom, dependencies: { d
               </S.ListItem>
             ))
           ) : (
-            <div className="dp-msj-error bounceIn">
+            <BoxMessage className="dp-msj-error bounceIn">
```
Comment (cbernat) [inline src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js]: This styled component should be named with prefix like `S.BoxMessage` I think we agreed on that some time ago.

Comment (fcoronelmakingsense) [reply src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js]: Same

File: src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js [hunk 4]
```diff
@@ -3,6 +3,7 @@ import { InjectAppServices } from '../../../services/pure-di';
 import { FormattedMessage, FormattedNumber } from 'react-intl';
 import Loading from '../../Loading/Loading';
 import * as S from './ReportsTrafficSources.styles';
+import { BoxMessage } from '../../styles/messages';
```
Comment (fcoronelmakingsense) [inline src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js]: import * as S from '../../styles/messages';

Comment (andresmoschini) [reply src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js]: There is already an `S`:

image

We have talked with @GusBaamonde that in the long term we need more reusable styles, for example, there should not be `ReportsTrafficSources.styles`, I mean styles specific for traffic sources.

**Pull request merged**